docs/components/sidecar.md

```diff
@@ -181,3 +183,36 @@ Flags:
       --version                  Show application version.
 
 ```
+
+## Configuration
+
+### Prometheus HTTP client
+
+The `--prometheus.http-client` and `--prometheus.http-client-file` flags allows the configuration for Prometheus HTTP client.
```

Comment:
It's worth mentioning that the 1st flag expects the YAML file contents and the 2nd expects a path to the YAML file.

Reply:
done!